apps/meteor/app/api/server/v1/billing.ts

```diff
@@ -0,0 +1,33 @@
+import { getCheckoutUrl } from '../../../cloud/server/functions/getCheckoutUrl';
+import { getWorkspaceAccessTokenWithScope } from '../../../cloud/server/functions/getWorkspaceAccessTokenWithScope';
+import { getURL } from '../../../utils/server/getURL';
+import { API } from '../api';
+
+API.v1.addRoute(
+	'billing.checkoutUrl',
+	{ authRequired: true, permissionsRequired: ['manage-cloud'] },
+	{
+		async get() {
+			const { token } = await getWorkspaceAccessTokenWithScope('workspace:billing');
+			if (!token) {
+				return API.v1.unauthorized();
+			}
+
+			const subscriptionURL =
+				process.env.NODE_ENV !== 'production' ? 'http://localhost:3000/admin/subscription/' : getURL('admin/subscription');
```

Comment:
As @sampaiodiego already mentioned, I think we don't need this, what we need is to provide the parameter `full` to the `getURL` function this way the function will use the Site_URL as the root URL. @sampaiodiego please correct me if I am wrong.